### DISCUSSION TITLE
Add SPI documentation shields.io badges

### DISCUSSION THREAD

**Discussion by hassila** [SwiftPackageIndex/SwiftPackageIndex-Server#2286] [Ideas]
Would be neat with a SPI shields.io badge for the documentation that links to the relevant documentation page at SPI, WDYT?

**Comment by daveverwer** [comment]
Yep, we agree! It's being discussed here: #2119

I think the consensus is that what you have proposed here, a simple badge with a link to the documentation is exactly what we have settled on as how to implement this. I'll lock this one, though, to consolidate the threads.
